src/java/htsjdk/samtools/CRAMFileWriter.java

```diff
@@ -46,8 +46,9 @@
 
 @SuppressWarnings("UnusedDeclaration")
 public class CRAMFileWriter extends SAMFileWriterImpl {
-    private static final int REF_SEQ_INDEX_NOT_INITIALIZED = -2;
+    private static final int REF_SEQ_INDEX_NOT_INITIALIZED = -3;
     static int DEFAULT_RECORDS_PER_SLICE = 10000;
+    static int SWITCH_TO_MULTIREF_IF_MORE_THAN = 1000;
```

Comment:
Its kind of hard to tell what the significance of this number is. It looks like it somehow affects alignment span in downstream code, but it would be helpful to have a comment somewhere describing why we would want to force a switch to multi-ref in the case where there are > 1000 records, even if they have the same same reference index (if I have that right).